examples/jsm/exporters/GLTFExporter.js

```diff
@@ -1852,7 +1852,17 @@ class GLTFWriter {
 				! ( array instanceof Uint8Array ) ) {
 
 				console.warn( 'GLTFExporter: Attribute "skinIndex" converted to type UNSIGNED_SHORT.' );
-				modifiedAttribute = new BufferAttribute( new Uint16Array( array ), attribute.itemSize, attribute.normalized );
+				modifiedAttribute = new BufferAttribute( new Uint16Array( attribute.count * attribute.itemSize ), attribute.itemSize, attribute.normalized );
+
+				for ( let i = 0; i < attribute.count; i ++ ) {
+
+					for ( let j = 0; j < attribute.itemSize; j ++ ) {
+
+						modifiedAttribute.setComponent( i, j, attribute.getComponent( i, j ) );
+
+					}
+
+				}
```

Comment:
To match the 'else if' block below, do you mind moving this loop to a new utility function like `GLTFExporter.Utils.toUint16BufferAttribute( attribute )`? Or the existing utility function could be extended and renamed as:

```javascript
modifiedAttribute = GLTFExporter.Utils.toTypedBufferAttribute( attribute, Uint16Array );
```